src/views/Home/components/Milestones/index.scss

```diff
@@ -1,128 +1,130 @@
       width: 374px;
-      height: 87px;
+      height: unset;
       border: 0;
-      justify-content: left;
-    }
-    &__header-text {
-      margin: 0;
-      padding: 0;
-      height: 67px;
-      width: 354px;
-      border-radius: 0%;
-      border-bottom-right-radius: 33.5px;
-      border-top-right-radius: 33.5px;
-      @include responsiveness.font($responsive: 4.1vw, $min: 25px, $max: 35px);
+
+      @include responsiveness.screen(smallest, phone) {
+        width: 80%;
+      }
     }
+
     &__header-border {
-      position: relative;
-      bottom: 50px;
-      left: 210px;
       display: block;
+      position: absolute;
+      bottom: -15px;
+      right: -15px;
+    }
+
+    &__header-ellipse {
+      display: none;
     }
+
     &__achievements {
-      .milestone {
-        &__image {
-          display: flex;
-          align-items: center;
-          justify-content: center;
-          min-width: 200px;
-        }
-        img {
-          width: 75%;
-          height: 60%;
-        }
-        &__box {
-          width: 180px;
-          justify-content: space-evenly;
-        }
-        &__stat {
-          line-height: 30px;
-          @include responsiveness.font(
-            $responsive: 4.1vw,
-            $min: 35px,
-            $max: 50px
-          );
-        }
-        &__name {
-          line-height: 25px;
-          @include responsiveness.font(
-            $responsive: 4.1vw,
-            $min: 21px,
-            $max: 28px
-          );
+      padding: 0 10px;
+    }
+
+    &__milestone {
+      &__image {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        min-width: 200px;
+      }
+
+      &__box {
+        justify-content: space-evenly;
+        padding: 10px;
+        width: 195px;
+        border-radius: 22px;
+
+        @include responsiveness.screen(smallest, phone) {
+          width: 165px;
         }
       }
-    }
-  }
-}
 
-@mixin milestone-small {
-  .milestones-component__header-border {
-    position: relative;
-    display: block;
-  }
-  .milestones-component__header-circle {
-    min-width: 240px;
-    max-width: 320px;
-    width: 80%;
-    .milestones-component__header-text {
-      width: 304px;
+      &__stat {
+        @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 50px);
+      }
+
+      &__name {
+        @include responsiveness.font($responsive: 4.1vw, $min: 21px, $max: 28px);
+      }
     }
   }
 }
 
 .milestones-component {
+  margin-top: -1px;
   padding: 14px 0 40px 0;
-  background: linear-gradient(
-    var(--color-light-pink),
-    var(--color-lavender),
-    var(--color-sky-blue)
-  );
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-  flex-wrap: wrap;
+  background: linear-gradient(var(--color-light-pink), var(--color-lavender), var(--color-sky-blue));
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 50px auto;
```

Comment:
On bigger screens (1400-2000) the spacing between the milestone-header and the milestone-achievements doesn't space around as cleanly so we might want to include a justify-content: space-around. There is another bug available for having this component scale for ~2000 -> 2500 (mainly because only this component is a little awkward on widescreens, so we can save tackle issue for that ticket or we can try fixing it here.

Reply:
Ah, you had that before but I was trying to match the design for desktop. I just pushed a change that'll use `justify-content: space-around` for bigger screens, so let me know how that looks. 

I can also keep the original version, but I feel like the `milestone-header` and `milestone-achievements` are too far away on desktop (~1440px).

Reply:
I think it looks okay, we should probably send a picture of both to design to see which one they like more and confirm from there.

Reply:
We can probably build off this PR for scaling the screen size for larger screens but this is a good refactor!